whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/DomainAuthSpec.groovy

```diff
@@ -3207,4 +3219,42 @@ class DomainAuthSpec extends BaseQueryUpdateSpec {
         queryObjectNotFound("-rGBT domain 4.4.e164.arpa", "domain", "4.4.e164.arpa")
     }
 
+    def "create overlapping domain"() {
+        given:
+        syncUpdate(getTransient("ALLOC-PA-LOW-DOM-R") + "password: hm\npassword: owner3")
+        queryObject("-r -T inetnum 193.0.0.0 - 193.255.255.255", "inetnum", "193.0.0.0 - 193.255.255.255")
+        syncUpdate(getTransient("OVERLAP-DOMAIN") + "override: denis,override1")
+        ipTreeUpdater.rebuild();
+
+        when:
+        def message = syncUpdate("""\
+                Domain:         191-193.0.0.193.in-addr.arpa
+                descr:          reverse domain
+                admin-c:        TP1-TEST
+                tech-c:         TP1-TEST
+                zone-c:         TP1-TEST
+                nserver:        pri.authdns.ripe.net
+                nserver:        ns3.nic.fr
+                mnt-by:         owner-MNT
+                source:         TEST
+
+                password:   lir2
+                password:   owner
+                """.stripIndent()
+        )
+
+        then:
+        def ack = new AckResponse("", message)
+
+        ack.summary.nrFound == 1
+        ack.summary.assertSuccess(0, 0, 0, 0, 0)
+        ack.summary.assertErrors(1, 1, 0, 0)
+        ack.countErrorWarnInfo(1, 0, 0)
+        ack.errors.any { it.operation == "Create" && it.key == "[domain] 191-193.0.0.193.in-addr.arpa" }
+        ack.errorMessagesFor("Create", "[domain] 191-193.0.0.193.in-addr.arpa") ==
```

Comment:
We need to report the overlapping reverse delegation and not the (forward) IPv4 prefix, otherwise it's really confusing for the user - they need to reverse the prefix again to identify the overlapping domain

Reply:
Done.